### PR TITLE
Update ci to pull app-bitcoin on master for litecoin build.

### PR DESCRIPTION
**Pull request by agrojean-ledger**

### CODE REVIEW

File: test/python/apps/bitcoin.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ class Command:
 
 
 class BitcoinClient:
-    CLA = 0xE0
+    CLA = 0xE1
```
Comment (agrojean-ledger) [inline test/python/apps/bitcoin.py:62]: Why this change ? @lpascal-ledger

Comment (lpascal-ledger) [reply test/python/apps/bitcoin.py:62]: `0xE0` was the CLA for the old Bitcoin application. The new one is now `0xE1`. This does not impact anything as currently there are no communication directly with the BTC application in the Python tests. It's more a reminder for possible future use (although we would probably use [the official client](https://pypi.org/project/ledger-bitcoin/) instead of this code).

Comment (bigspider) [reply test/python/apps/bitcoin.py:62]: It appears that `BitcoinClient` is not actually used, so probably better to remove it?
Just changing `E0` to `E1` wouldn't fix the class, as the protocol is completely different anyway.

Comment (lpascal-ledger) [reply test/python/apps/bitcoin.py:62]: Indeed, it's dead code, I'll remove it.

**Review by bigspider — APPROVED** [review]
LGTM
